client/my-sites/plugins/main.jsx

```diff
@@ -431,15 +433,17 @@ export class PluginsMain extends Component {
 				<QueryJetpackPlugins siteIds={ this.props.siteIds } />
 				<QuerySiteFeatures siteIds={ this.props.siteIds } />
 				{ this.renderPageViewTracking() }
-				<FixedNavigationHeader
-					className="plugins__page-heading"
-					navigationItems={ this.getNavigationItems() }
-				>
-					<div className="plugins__main-buttons">
-						{ this.renderAddPluginButton() }
-						{ this.renderUploadPluginButton( this.state.isMobile ) }
-					</div>
-				</FixedNavigationHeader>
+				{ ! this.props.isJetpackCloud && (
```

Comment:
What do you think about changing the name of this prop from `isJetpackCloud` to `shouldDisplayNavigationHeader` (or something similar)?

If the only purpose of that prop is hiding/showing the navigation header (since now we don't need a different base path for the URL), a more specific name for the prop will result in a more generic approach.

By the way, I don't have a strong feeling about this.

Reply:
I have made the changes and using `shouldDisplayNavigationHeader` in the child component so that it is easy to understand, but still keeping the props from the component as `isJetpackCloud` in the parent component since we will need it for other changes in the upcoming PRs